pyscf/pbc/cc/kccsd.py

```diff
@@ -391,7 +391,9 @@ def init_amps(self, eris):
         return self.emp2, t1, t2
 
     def ccsd(self, t1=None, t2=None, eris=None, **kwargs):
-        if eris is None: eris = self.ao2mo(self.mo_coeff)
+        if eris is None:
+            eris = self.ao2mo(self.mo_coeff)
+            self.e_hf = self.get_e_hf()
```

Comment:
self.e_hf may not be created if eris is specified

Reply:
I've deleted the self.e_hf evaluation here.